common/src/main/java/dev/lavalink/youtube/clients/skeleton/NonMusicClient.java

```diff
@@ -87,8 +87,8 @@ protected JsonBrowser loadTrackInfoFromInnertube(@NotNull YoutubeAudioSourceMana
 
         ClientConfig config = getBaseClientConfig(httpInterface);
 
-        if (status == null) {
-            // Only add embed info if the status is not NON_EMBEDDABLE.
+        if (status == null || isEmbedded()) {
```

Comment:
Remove this change. This will cause recursion issues, and embed clients will still have the clientScreen and other attributes set on the first request anyway.

Reply:
I'll do this but I'm also noting here that it's adding `clientScreen: EMBED` to **all** non-music clients, regardless of if they're actually embedded or not. Maybe it helps playback currently, maybe not. No idea. Maybe in future they'll block embedding on non-embedded clients.